metadata_tools/index_support.py

```
@@ -399,17 +391,17 @@ def _format_column(column_stub, value, count=None):
         except TypeError:
-            logger.warn("Invalid format: %s %s %s" % (name, value, format))
+            logger.warning("Invalid format: %s %s %s" % (name, value, format))
             result = width * "*"
 
         if len(result) > width:
-            logger.warn("No second format: %s %s %s %s" % (name, value, format, result))
+            logger.warning("No second format: %s %s %s %s" % (name, value, format, result))
 
         # Validate the formatted value
         try:
             _ = eval(result)
```

Ruff (0.14.10)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)